bin/remixdCLI.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+const path = require('path');
+const app = require('express')();
+const ws = require('express-ws')(app);
+const cors = require('cors');
+//const server = require('http').createServer(app)
+//const io = require('socket.io')(server)
+const bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+
+
+// Middleware
+app.use(bodyParser.urlencoded({extended: true}))
+app.use(bodyParser.json());
+app.use(cors({origin: true}));
+app.options('*', cors());
+
+//Commander part
+let program = require('commander');
```

Comment:
As far as I understand this code `program` is not reassigned (just mutated), so you should be able to use `const`

Reply:
Also fixed